### ISSUE TITLE
When dropping a file on Gmail in OSX, the screen turns black

### ISSUE THREAD

**Issue by metaregistrar** [meetfranz/franz#72]
### Expected Behavior

I want to drop an image in an open gmail message, to attach an image to my mail.

### Current Behavior

When i drop a file in gmail (to put an image in my mail) the screen turns completely black. No way to go back to the previous screen. I had to stop and restart franz to make my mail work again.

### Steps to Reproduce (for bugs)

1. Create a gmail service and login
2. Create a new message
3. Drag and drop an image to the gmail window
4.

### Your Environment

* Franz Version used: 5.0.0-beta.10
* Operating System and version: macOS high sierra 10.13

**Comment by iiiaxx** [comment]
I'm experiencing a similar behaviour using Slack service. To reproduce it I don't need to drop a file though, the interface goes black randomly after a while (>20 minutes of use). Other services work fine. Should I create a separate issue for it?

![screen shot 2017-10-26 at 10 44 13](https://user-images.githubusercontent.com/19621044/32046408-e142540e-ba3a-11e7-8ba2-d181c3d6ad55.png)

### Your Environment

* Franz Version used: 5.0.0-beta.11
* Operating System and version: macOS high sierra 10.12.5

**Comment by metaregistrar** [comment]
I am using 2 different Slack services with the current version of Franz, both have different logins - no problems at all, it keeps on functioning througout the day.

**Comment by metaregistrar** [comment]
I found out that you not even have to drop the file. If you drag a file to any other window and move over the Franz window somehow (without dropping it) the window turns black.

**Comment by iiiaxx** [comment]
Yes, I confirm I'm having the same issue as @metaregistrar. the interface goes black, even without dropping the file in the app 😕

**Comment by dannyqiu** [comment]
Might be related to https://github.com/electron/electron/issues/9341

**Comment by simondahla** [comment]
I'm having this issue as well. Both for Hangouts and for Slack.

**Comment by spersico** [comment]
I'm having this issue in Whatsapp and Telegram, so i don't think its about the plugin implementation:

It happens when draggin a file into an **non-active** chat:
I'm in group chat A, i drag&drop an image or whatever in chatGroup B, the window turns black.

**Comment by adlk** [comment]
Thanks for your reports! I need to do a few tests with different electron versions to figure out if we can sail around this issue somehow until there is an official fix.

**Comment by krystofhonsu** [comment]
Confirmed this issue also for Whatsapp and Facebook Messenger. In native Whatsapp Mac App this functionality works.

Environment: Mac OS 10.12.5, Franz Version 5.0.0-beta.13 (5.0.0-beta.13.316)

**Comment by fagiani** [comment]
I confirm I'm experiencing this very same behavior on Telegram on a Mac 10.12.6 and Franz 5.0.0-beta.13 as well.

**Comment by adlk** [comment]
Yes, it happens on all services. While there is no instant fix for it, we have added a crash handler to easily recover the service. This will be shipped with beta 14.

I hope that this issue is addressed with one of the next electron/chromium updates.

**Comment by BrianGilbert** [comment]
Pasting from message I posted in slack 23 days ago:

I think I figured the file dragging bug, if you drag a file across the services bar onto the window and then off again without dropping the main area of the screen will go black..  This doesn't happen if you don't drag over the services bar. (this was confirmed by others)

Once the main screen area of service has gone black you have to use refresh to recover it.. you can switch to another service and that one will still be working but switching back to the original one it will still be black

**Comment by ksn135** [comment]
Franz Version 5.0.0-beta.14 (5.0.0-beta.14.451) and macOS Sierra 10.12.6 (16G29).
Same (black screen) happens to me with Telegram service. Drag'n'drop doesn't work.
Using Developer Tools it's show me several JS errors:
![2017-12-01_20-36-33](https://user-images.githubusercontent.com/230304/33495584-0b1923e8-d6d8-11e7-93b2-fb37a348c54b.png)

**Comment by diego-sorribas** [comment]
I had the same issue dragging an html to slack channel. It's render me the html on the screen.

**Comment by BrianGilbert** [comment]
This is an upstream bug with react, @adlk is waiting for the fix there.

**Comment by haraldox** [comment]
this now seems to work for files dropped in Franz 5.0.0-beta.15 on macOS (tested for WhatsApp, slack, messenger and gmail) ... please test your services again and report! 🙏 

Any Windows users experience this issue?

HTML images drag'n'dropped from Chrome into WhatsApp don't crash the service either ... but we're not sure yet if we officially support that in Franz

**Comment by spersico** [comment]
Still Happening on Franz 5.0.0-beta. 15, on windows 10.
Tested on whatsapp, discord and telegram, and it only happens when dragging over the service selector
![2018-01-15_21-53-01](https://user-images.githubusercontent.com/10502605/34967019-9ec034a4-fa3e-11e7-870e-b4041e8371fa.png)

**Comment by BrianGilbert** [comment]
yes still occurring for me as well
